build.gradle.kts

```diff
@@ -329,9 +329,9 @@ tasks.register("python38PostCommit") {
 
 tasks.register("python39PostCommit") {
   dependsOn(":sdks:python:test-suites:dataflow:py39:postCommitIT")
-  dependsOn(":sdks:python:test-suites:direct:py39:postCommitIT")
-  dependsOn(":sdks:python:test-suites:direct:py39:hdfsIntegrationTest")
-  dependsOn(":sdks:python:test-suites:portable:py39:postCommitPy39")
+  // dependsOn(":sdks:python:test-suites:direct:py39:postCommitIT")
```

Comment:
In general, I think you should instead do 1 of 2 things. Either:
1. Just delete these lines.
2. above the commented out section put in a comment that says:
```
# TODO(issueReference): Tests temporarily disabled because ...
```

Reply:
Oh this is only for testing why the UI not showing few tests. It's not ready for review yet

Reply:
FWIW, you can keep the bot from notifying people by either:
1. Marking as draft
2. Prefixing the PR title with `WIP`
3. Commenting `stop reviewer notifications`